examples/complex_fluids/ex0/input2d

```diff
@@ -33,8 +31,8 @@ CONVECTIVE_OP_TYPE = "PPM"                // convective differencing discretizat
 CONVECTIVE_FORM    = "ADVECTIVE"          // how to compute the convective terms
 NORMALIZE_PRESSURE = TRUE                 // whether to explicitly force the pressure to have mean zero
 CFL_MAX            = 0.3                  // maximum CFL number
-DT_MAX = 0.1/N/0.1
-END_TIME = 4.0
+DT_MAX             = 0.01/N               // maximum time step size
```

Comment:
Do we support AMR with complex fluids? If so I'd like to use `NFINEST` here instead of `N`.